### PR TITLE
January TMIS newsletter

### PR DESCRIPTION
**Pull request by MarrelleBailey**
Can we make this the newsletter to start the new newsletter

### CODE REVIEW

**Review by MarrelleBailey — COMMENTED** [review]
Just needed to add the links

**Review by MarrelleBailey — COMMENTED** [review]
I believe everything should be up to date now, sorry for all the changes

File: _posts/newsletter/next.md [hunk 1]
```diff
@@ -5,28 +5,37 @@ permalink: /newsletter/2020-01-07
...
+
+## Discussed: This Month in Solid 
+
+Hello and welcome to another edition of This Month in Solid!
+
+[Solid](http://solidproject.org/) is a set of open specifications, built on existing open standards, that describes how to build applications so that users can conveniently switch between data storage providers and application providers and take the data generated along.
+
+This is a monthly summary of Solid progress.
+
+Want something mentioned? Contact the Solid team at info@solidproject.org. This Month in Solid is openly developed [on GitHub](https://github.com/solid/solidproject.org/blob/staging/_posts/newsletter/next.md). If you find any errors in this month’s issue, please [submit a pull request](https://github.com/solid/solidproject.org/pulls).
```
Comment (Vinnl) [inline _posts/newsletter/next.md:17]: We no longer use a separate `staging` area:

```suggestion
Want something mentioned? Contact the Solid team at info@solidproject.org. This Month in Solid is openly developed [on GitHub](https://github.com/solid/solidproject.org/blob/master/_posts/newsletter/next.md). If you find any errors in this month’s issue, please [submit a pull request](https://github.com/solid/solidproject.org/pulls).
```

File: _posts/newsletter/next.md [hunk 2]
```diff
@@ -5,28 +5,37 @@ permalink: /newsletter/2020-01-07
...
+
+Next we heard from Ruben Verborgh, who used the Community Solid Server to demonstrate how his smart lights could be controlled via his Pod. You can see the code behind the demo [here](https://github.com/RubenVerborgh/solid-hue). We also heard from Osmar (Oz) Olivo about Inrupt’s Enterprise Solid Server, reflections on their growth and what they are working toward this year. Last, but very much so not least, we heard from Sarven Capadisli, who spoke about the specification updates on behalf of the W3C Solid Community Group. He demonstrated all the growth they have made to build our current foundation. He also mentioned that the panels for discussing the specs are open to anyone. There were many resources shared during the Solid World, but no worries if you miss them we have the links and resources available [here](https://solidproject.org/events).
```
Comment (Vinnl) [inline _posts/newsletter/next.md:30]: I think the list of resources is separate from the description of Sarven's talk, right?

```suggestion
Next we heard from Ruben Verborgh, who used the Community Solid Server to demonstrate how his smart lights could be controlled via his Pod. You can see the code behind the demo [here](https://github.com/RubenVerborgh/solid-hue). We also heard from Osmar (Oz) Olivo about Inrupt’s Enterprise Solid Server, reflections on their growth and what they are working toward this year. Last, but very much so not least, we heard from Sarven Capadisli, who spoke about the specification updates on behalf of the W3C Solid Community Group. He demonstrated all the growth they have made to build our current foundation. He also mentioned that the panels for discussing the specs are open to anyone.

There were many resources shared during the Solid World, but no worries if you missed them: we have the links and resources available [here](https://solidproject.org/events). 
```

**Review by Vinnl — APPROVED** [review]
Looks good to me, I added just one suggestion for you to consider (and fixed one link).

Do you also want to create `2021-01-07-this-month-in-solid.md`, or shall I do that?

File: _posts/newsletter/next.md (hunk 2 above)
Comment (MarrelleBailey) [reply _posts/newsletter/next.md:30]: Yea, they are separate

Comment (MarrelleBailey) [reply _posts/newsletter/next.md:30]: Oh also I can create the PR for Jan

Comment (MarrelleBailey) [reply _posts/newsletter/next.md:30]: Actually can you create the Jan newsletter it will be faster that way

Comment (Vinnl) [reply _posts/newsletter/next.md]: Will do.

**Pull request merged**